engine/src/main/kotlin/world/gregs/voidps/engine/inv/InventoryOperations.kt

```diff
@@ -28,6 +30,8 @@ fun Inventory.shift(fromIndex: Int, toIndex: Int) = transaction { shift(fromInde
 
 fun Inventory.add(id: String, amount: Int = 1) = transaction { add(id, amount) }
 
+fun Inventory.addAll(vararg items: Item) = transaction { items.forEach { item -> add(item.id, item.amount) } }
+
```

Comment:
```suggestion
```
Doesn't look like it's used anywhere atm, would rather it be left out until needed, also for scenario's that do need it using `transaction {}` directly is probably better.